custom/conf/app.ini.sample

```diff
@@ -332,6 +332,9 @@ MIN_PASSWORD_LENGTH = 6
 IMPORT_LOCAL_PATHS = false
 ; Set to true to prevent all users (including admin) from creating custom git hooks
 DISABLE_GIT_HOOKS = false
+;Comma separated list of character classes required to pass minimum complexity.
+;If left empty or no valid values are specified, the default values (`lower,upper,digit,spec`) will be used.
+PASSWORD_COMPLEXITY = lower,upper,digit,spec
```

Comment:
I think we need some way of actually switching this off formally as it will be extremely irritating for testing to have to have complex passwords.

Reply:
Something like:
```ini
PASSWORD_COMPLEXITY = off
```

Reply:
(Sorry to just throw this in right before it gets merged.)

Reply:
@zeripath for testing environment you can use:
```
PASSWORD_COMPLEXITY = lower
```

Reply:
OK, looking again at the code I think if you set `PASSWORD_COMPLEXITY=,` it should turn it off.

Reply:
I'm gonna dismiss my review.